### DISCUSSION TITLE
[TOPAS validation]: The neutron dose induction based on the kerma approximation in proton radiation therapy

### DISCUSSION THREAD

**Discussion by Pornpapas** [OpenTOPAS/OpenTOPAS#75] [Validation] [answered]
### Contact Details

-

### Describe your simulation result

Dear Developer

I would like to calculate or score the neutron dose induction based on the kerma approximation in proton radiation therapy. Could you advise me on the selection of modules and settings

best regard.

### Version

4.0

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Attestation

- [X] I testify that I have fully read the bug reporting guideline beforehand.

**Comment by JoseRamosMendez** [comment, marked as answer]
Hi there,
Take a look at this [example](https://github.com/OpenTOPAS/OpenTOPAS/blob/e7aefdc261a0bddd7740922150cc19a49aec62a4/examples/VarianceReduction/AmbientDoseEquivalent.txt), further described [here](https://opentopas.readthedocs.io/en/latest/parameters/scoring/volume.html#ambientdoseequivalent). It uses such approximation (track-length estimator). You could use the provided fluence-to-dose conversion factors or use your own.
Best.
